Aircoookie/WLED
# Add PlatformIO and npm install to devcontainer

This adds the following two items to the devcontainer initialization:

1. Installs the PlatformIO extension automatically since it's required to do anything useful
2. Runs `npm install`

With these changes a devcontainer created directly with a cloned WLED repo can do a build with no additional required steps. 

To test:

1. Make sure you have the VSCode - Remote extensions installed
2. Inside VSCode run `Remote-Containers: Clone repository in container volume...`
3. Point to `https://github.com/danecreekphotography/WLED/tree/devcontainer-updates`

Once the volume is created hit the checkmark and voila! A build.
